stable/percona-xtradb-cluster/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 name: percona-xtradb-cluster
-version: 0.1.5
+version: 0.1.6
```

Comment:
here @obedmr instead of `0.1.6` change to `0.2.0`.

Actually, realize now this chart is under stable then I would set the version to `1.0.0`. We had a call this week and was discussed to all charts in a stable folder should be at least `1.0.0` and not `0.x.x`

Reply:
So, you agree with setting as  `1.0.0`?

Reply:
from my side, yes 😄

Reply:
@paulczar, @davidkarlsen  any objection?

Reply:
@cpanato do you have a link to the minutes or notes about that agreement? just to put it as a reference in the commit message.

Reply:
https://docs.google.com/document/d/1h6UTTuNRbFI81higrN3JUV2XxyzqqVjZET4Xz4WTR-8/edit#heading=h.57pbxthvt6k3 (Enforcing semver section)

Reply:
@cpanato thanks.